package.json

```diff
@@ -64,7 +64,7 @@
     "lodash.union": "^4.6.0",
     "lodash.values": "^4.3.0",
     "object-hash": "^3.0.0",
-    "packageurl-js": "^1.0.0",
+    "packageurl-js": "1.2.0",
```

Comment:
Suggestion: 

The library should stable version instead of pinning to a specific version to prevent changes of having multiple copies of packageurl-js in the final executable.

```diff
-    "packageurl-js": "1.2.0",
+    "packageurl-js": "<= 1.2.0",
```

in the future:

```diff
-    "packageurl-js": "<= 1.2.0",
+    "packageurl-js": "<= 1.2.0 || >= 1.2.2",
```